lib/iron_bank/resources/product_rate_plan.rb

```diff
@@ -9,7 +9,11 @@ class ProductRatePlan < Resource
       # NOTE: Zuora doesn't let us query for more than one product rate plan
       #       `ActiveCurrencies` at a time
       def self.excluded_fields
-        super + %w[ActiveCurrencies]
+        super + separate_query_fields
+      end
+
+      def self.separate_query_fields
```

Comment:
WDYTA moving to a private constant (eg., `SINGLE_RESOURCE_QUERY_FIELDS`)?

Reply:
Good point for the naming, but won't be using a constant in just that file since it would require introspection to check if the class defines it.